### ISSUE TITLE
proposal: percentage values for font-size

### ISSUE THREAD

**Issue by alixander** [terrastruct/d2#709]
https://github.com/terrastruct/d2/issues/429#issuecomment-1399573981

in consideration.

I can't really think of a scenario where i want to set all these percentages to the way i want, and then later increase or decrease the base. 

the value of this on the web is accessibility, like if the viewer has zoom up or something, then the page scales. but that doesn't work for D2, because the fonts have to be pre-measured, or else they'll change the dimensions of the shapes which changes layout or causes collisions.
